packages/main/cypress/specs/Dialog.cy.ts

```diff
@@ -0,0 +1,413 @@
+import { html } from "lit";
+import "../../src/Dialog.js";
+import "../../src/Button.js";
+import "../../src/Input.js";
+
+describe("Dialog", () => {
+	describe("Dialog general interaction", () => {
+		beforeEach(() => {
+			cy.mount(html`
+				<ui5-button id="btnOpenDialog">Open Stretched Dialog</ui5-button>
```

Comment:
why do we need button and event listener to open the dialog?

it can be achieved using invoke or just render it initially.